### PR TITLE
Filter icons.svg comment from output

### PR DESCRIPTION
**Pull request by distantnative**

### CODE REVIEW

File: src/Panel/Assets.php [hunk 1]
```diff
@@ -151,7 +151,9 @@ public function icons(): string
 	{
 		$dir  = $this->kirby->root('panel') . '/';
 		$dir .= $this->dev ? 'public' : 'dist';
-		return F::read($dir . '/img/icons.svg');
+		$icons = F::read($dir . '/img/icons.svg');
+		$icons = preg_replace('/<!--(.|\s)*?-->/', '', $icons);
```
Comment (lukasbestle) [inline src/Panel/Assets.php:155]: Would this work as well (better performance)?

```suggestion
		$icons = Str::after($icons, '-->');
```

Comment (distantnative) [reply src/Panel/Assets.php:155]: Isn't this making too many assumptions? What if we ever add something before the comment in the SVG file but don't remember this code part?

Comment (lukasbestle) [reply src/Panel/Assets.php:155]: TBH I think this is rather theoretical. The purpose of this PR is to remove the license comment and I don't see a reason why it needs to be anywhere else than at the beginning of the file.

If we do ever change that for whatever reason in the future, we can change this line as well. We will see when it breaks and can track down that this line is responsible.

Comment (bastianallgeier) [reply src/Panel/Assets.php:155]: If preg_replace does not have any noticeable performance hit with the large icon file, I would use the more robust solution.

**Comment by bastianallgeier** [comment]
Let's merge it like this. It's easy enough to optimize later if needed.

**Pull request merged**